packages/block-editor/src/components/iframe/index.js

```diff
@@ -259,7 +260,7 @@ function Iframe( {
 				// mode. Also preload the styles to avoid a flash of unstyled
 				// content.
 				src={ src }
-				title={ __( 'Editor canvas' ) }
```

Comment:
Turns out this is actually an accessibility violation. This isn't a rule I agree with considering the iFrame is wrapped in a `role="region"` with a valid `aria-label`.

@afercia You think this is something we can get away with given the situation?

Having a region labeled as Editor Content followed by the iFrame labeled as Editor Canvas is very verbose and not helpful. The other option, remove the `aria-label` from this region and the title should be good enough since it is the only element?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role

Regions seem to require accessible labels so really struggling here as to how labeling should:
A. Not be annoying.
B. Not be an accessibility violation from the lack of labeling.